src/annotator/test/sidebar-test.js

```diff
@@ -724,9 +738,19 @@ describe('Sidebar', () => {
 
     it('does not have the BucketBar if an external container is provided', () => {
       const sidebar = createSidebar({
-        externalContainerSelector: '.' + EXTERNAL_CONTAINER_SELECTOR,
+        externalContainerSelector: `.${EXTERNAL_CONTAINER_SELECTOR}`,
       });
       assert.isUndefined(sidebar.plugins.BucketBar);
     });
+
+    it('disables shadow DOM', () => {
```

Comment:
```suggestion
    it('disables shadow DOM if `disableShadowSidebar` flag is set', () => {
```

This just makes the test description more obvious.